bundles/org.openhab.binding.publictransportswitzerland/src/main/java/org/openhab/binding/publictransportswitzerland/internal/stationboard/PublicTransportSwitzerlandStationboardHandler.java

```diff
@@ -0,0 +1,187 @@
+            String requestUrl = BASE_URL + "stationboard?station=" + escapedStation + fieldFilters;
+
+            String response = HttpUtil.executeUrl("GET", requestUrl, 10_000);
+            logger.debug("Got response from API: {}", response);
+
+            JsonElement jsonObject = new JsonParser().parse(response);
+
+            updateChannels(jsonObject);
+            updateStatus(ThingStatus.ONLINE);
+        } catch (Exception e) {
+            logger.warn("Unable to fetch stationboard data", e);
+
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+            updateState(CHANNEL_TSV, new StringType("No data available"));
+        }
+    }
+
+    private static String createFilterForFields(String... fields) {
+        return Arrays.stream(fields).map((field) -> "&fields[]=" + field).collect(Collectors.joining());
+    }
+
+    private void updateChannels(JsonElement jsonObject) throws Exception {
+        JsonArray stationboard = jsonObject.getAsJsonObject().get("stationboard").getAsJsonArray();
+        createDynamicChannels(stationboard.size());
+
+        List<String> tsvRows = new ArrayList<>();
+
+        for (int i = 0; i < stationboard.size(); i++) {
+            JsonElement jsonElement = stationboard.get(i);
+
+            JsonObject departureObject = jsonElement.getAsJsonObject();
+            JsonObject stopObject = departureObject.get("stop").getAsJsonObject();
+
+            String category = departureObject.get("category").getAsString();
+            String number = departureObject.get("number").getAsString();
+            String destination = departureObject.get("to").getAsString();
+
+            JsonElement delayElement = departureObject.get("delay");
+            String delay = "";
+            if (delayElement != null) {
+                delay = delayElement.getAsString();
+            }
+
+            Long departureTime = stopObject.get("departureTimestamp").getAsLong();
+            String track = stopObject.get("platform").getAsString();
+
+            updateState(getChannelUIDForPosition(i), new StringType(formatDeparture(category, number, departureTime, destination, track, delay)));
+            tsvRows.add(String.join("\t", category, number, departureTime.toString(), destination, track, delay));
+        }
+
+        updateState(CHANNEL_TSV, new StringType(String.join("\n", tsvRows)));
+    }
+
+    private String formatDeparture(String category, String number, Long departureTimestamp, String destination, String track, @Nullable String delay) {
+        Date departureDate = new Date(departureTimestamp * 1000);
+
+        SimpleDateFormat timeFormat = new SimpleDateFormat("HH:mm");
+        String formattedDate = timeFormat.format(departureDate);
+
+        String train = number.startsWith(category) ? category : category + number;
+        String result = String.format("%s - %s %s - Pl. %s", formattedDate, train, destination, track);
```

Comment:
Pl is for platform ?

Reply:
In the official app they do it with Pl. as well. See below